src/domain_bridge/parse_domain_bridge_yaml_config.cpp

```diff
@@ -142,6 +142,34 @@ static QosOptions parse_qos_options(YAML::Node yaml_node, const std::string & fi
     }
   }
 
+  if (qos_node["liveliness"]) {
+    try {
+      auto liveliness_str = qos_node["liveliness"].as<std::string>();
+      if (0u == std::strcmp(liveliness_str.c_str(), "manual_by_topic")) {
+        options.liveliness(rclcpp::LivelinessPolicy::ManualByTopic);
+      } else if (0u == std::strcmp(liveliness_str.c_str(), "automatic")) {
+        options.liveliness(rclcpp::LivelinessPolicy::Automatic);
+      } else if (0u == std::strcmp(liveliness_str.c_str(), "system_default")) {
+        options.liveliness(rclcpp::LivelinessPolicy::SystemDefault);
+      } else {
+        throw YamlParsingError(
+                file_path, "livelines must be manual_by_topic/automatic/system_default");
```

Comment:
nit : Alignment

Reply:
this is what uncrustify wanted, if not it doesn't pass.
(see that CI is passing)